source/layouts/layout.slim

```diff
@@ -57,3 +57,8 @@ html.Theme itemscope='' itemtype='http://schema.org/WebPage' lang=data.site.lang
       try{Typekit.load();}catch(e){}
     = javascript_include_tag 'all'
     = yield_content :javascripts
+    javascript:
```

Comment:
actually `if production?` would be better. we don't want this in test/staging either